doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<ul><li>I would add a new subtitle headline here</li></ul><p> </p><ul><li>"Let's see if the model can find a masked word in an input sentece. As an Example we will use an instruction usually given to a patient when a radiographer is doing their chest X-ray."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Good idea. I never liked the appearance of the notebooks - just a wall of text. Headers should definitely make it look better.</p>

Reply:
<p>please also take a look at my phrasing related suggestions.</p>

Reply:
<p>I don't see anything wrong here.</p>

Reply:
<p>If you wanted to keep the sentence structure like this, it should be "Let's see if the model can fill <strong>in a </strong>masked word in an instruction usually given to a patient when<strong> a</strong> radiographer is doing<strong> their </strong>chest X-ray."</p><p>However i find that sentence a little convoluted and would split it in two as suggested in my previous comment.</p>

Reply:
<p>"<strong>For this example we will use</strong> an instruction usually given to a patient when a radiographer is doing<strong> their </strong>chest X-ray."</p><p> </p><p>Non-female people can  get chest x-rays also. </p>

Reply:
<p>Their is plural.</p>

Reply:
<p>they/them pronouns can be used for singular persons if their gender is not known.</p><p><a href="https://www.oxfordlearnersdictionaries.com/definition/english/their" target="_blank">https://www.oxfordlearnersdictionaries.com/definition/english/their</a></p><p><a href="https://en.wiktionary.org/wiki/they#English" target="_blank">https://en.wiktionary.org/wiki/they#English</a></p>